index.test.js

```diff
@@ -39,13 +54,13 @@ it('Can jit a single prop', async () => {
   await run(
 `a {
   color: var(--red);
-}`, 
+}`,
```

Comment:
can you shave these spacing edits from the PR please? will help focus the PR into only what's needing changed.